src/App/Pages/Accounts/LoginPageViewModel.cs

```diff
@@ -247,9 +259,8 @@ public async Task LogInAsync(bool showLoading = true, bool checkForExistingAccou
 
         private async Task MoreAsync()
         {
-            var buttons = IsEmailEnabled
-                ? new[] { AppResources.GetPasswordHint }
-                : new[] { AppResources.GetPasswordHint, AppResources.RemoveAccount };
+            var buttons = IsEmailEnabled || !SavedEmail
+                ? new[] { AppResources.GetPasswordHint } : new[] { AppResources.GetPasswordHint, AppResources.RemoveAccount };
```

Comment:
Consider line break like it was before to make it easier to read.

Reply:
Oops missed this

Reply:
fixed!